tools/linux_rms_sdk_for_cpp_setup.sh

```diff
@@ -1,12 +1,19 @@
-
 cd ../sdk
-qmake
-make clean
-make
-cd ../samples
-qmake
+qmake -recursive
 make clean
 make
+if [ $? -ne 0 ]; then
+  echo "sdk make failed."
+  exit 1
+fi
+# cd ../samples
```

Comment:
The code to make samples is commented out. Do we use another script for it?

Reply:
with all the file changes the sample app fails at the moment, lets make it a different issue to fix it